### DISCUSSION TITLE
Help me intergrate mongodb into my nextjs project

### DISCUSSION THREAD

**Discussion by prooftv** [vercel/next.js#49724] [Help]
### Summary

Hi, i need help, from an experienced or novice developer to intergrate mongodb, and mapbox into this purchased template : https://directory-v2.vercel.app/

Its been more than 2 years now, since am struggling to find someone to help me visualize my startup dream..

Please and thank you in advance,  the code is available on github..

### Example

https://github.com/prooftv/mrsalons/tree/main

**Comment by 808vita** [comment]
what were the things you tried and what didn't work ?

!and btw please properly check before committing changes to github. mongodb url with creds are visible  , update the credentials first !.

**Reply by 808vita** [reply]
also templates usually comes with documentations/ usage guides . Hope you got something like that from the vendor .
